File: Totalizer/Models/Event.cs
using System;

namespace Models
{
    public class Event
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public Team Team1 { get; set; }
        public Team Team2 { get; set; }
        public SportType SportType { get; set; }
        public string Status { get; set; }
    }
}

File: totalizer_UI/totalizer/src/app/events/Models/Event.ts
import { Team } from 'src/app/teams/Models/Team';
import { SportType } from 'src/app/sport-types/Models/SportType';

export class Event {
  Id: number;
  Date: Date;
  Team1: Team;
  Team2: Team;
  SportType: SportType;
  Status: string;
}
